src/util/opensmt_version.h.cmake.in

```diff
@@ -0,0 +1 @@
+#define OPENSMT_GIT_DESCRIPTION @OPENSMT_GIT_DESCRIPTION@
```

Comment:
I think we also should consider adding `opensmt_version.h`

It is autogenerated ofc, but it might be useful to have as information in the repo, since now code in the repository in `SMTConfig.cc` references `opensmt_version.h`

Reply:
I'm going to go with absolutes here.  I think we should never have autogenerated files in git.  It always causes problems.

Maybe this is actually a good argument for having it as a macro?

Reply:
Fixed